app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsApplicationDate.test.tsx.snap

```diff
@@ -0,0 +1,53 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Let's have a team convo about unit test approach / snapshot tests 👍

Reply:
We had a meeting today to discuss